### DISCUSSION TITLE
SOLVED System audio input device sample rate not supported?

### DISCUSSION THREAD

**Discussion by daniaprile** [jamulussoftware/jamulus#1332] [Technical Support & Help] [answered]
Hello! I recently installed the Jamulus update, and now it won't let me use my mic. When I try to use it, a pop up appears that says 
"The selected audio device could not be used because of the following error: Current system audio input device sample rate of 44100 Hz is not supported. Please open the Audio-MIDI-Setup in Applications->Utilities and try to set a sample rate of 4800 Hz"
I tried to do as the pop up said, but my microphone's format doesn't have any sort of drop down menu that will allow me to change the sample rate. (I have a Macbook air btw)

PLEASE HELP!! Thanks in advance!

**Comment by ann0see** [comment]
If you can’t set this to 48 kHz you‘ll probably not be able to use this device. 

Which devices do you have and what’s set as default microphone/output?

**Reply by daniaprile** [reply]
Thank you for your help! Turns out my mic has a fixed sample rate, but I emailed the company and they walked me through creating an aggregate device with my built in input, which allowed me to use the mic! Thank you again! (also the mic I am using is the FiFine T669)

**Reply by hoffie** [reply]
> but I emailed the company and they walked me through creating an aggregate device with my built in input, which allowed me to use the mic!

Great to hear, that sounds like great support by the company as well! Is this "aggregate device" stuff some standard thing on Macs? If not, could you maybe add the necessary steps in a comment here so that others who have the same problem can just try that?

**Reply by ann0see** [reply]
Yes. That's great company support, I'd say. "aggregate device" is a macOS only feature which we might want to document somewhere. Maybe macOS can resample 44.1kHz to 48kHz natively?

**Reply by daniaprile** [reply]
Thank you both for your advice! I'll add the steps for both window and mac that were provided to me.

**Comment by gilgongo** [comment]
>  I recently installed the Jamulus update, and now it won't let me use my mic.

Just checking - do you mean you have or have not been able to to use this mic with Jamulus in the past? If you **have** been able to use it a with a _previous_ version of Jamulus, then that might be a bug.

**Reply by daniaprile** [reply]
Hi thank you for your help! I was able to use my mic with jamulus in the past, so it very may well have been a bug. I was able to solve the problem though! Thank you again for your help!

**Comment by daniaprile** [comment, marked as answer]
SOLUTION
Windows: Install a asio4all application (link: http://www.asio4all.org/). Just enable your mic as the default input device in the asio4all, then choose the asio4all as io devices in Jamulus.

Mac: Open the midi audio application on your mac, then check your mic and "Built-in output" to create an "aggregated device" (please see the attachment). Just set your mic as input while your computer built-in speaker as output. Then choose the "aggregated device" as input devices in Jamulus.
